libs/components/charts/testing/src/modules/chart-bar/fixtures/chart-bar-harness-test.component.html

```
@@ -0,0 +1 @@
+TODO
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)